java-squid/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -44,4 +46,8 @@ public void checkEndOfExecution(CheckerContext context) {
     // By default do nothing
   }
 
+  public void checkEndOfExecutionPath(CheckerContext context, ConstraintManager constraintManager) {
+    // TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)